docs/rest-api-fundamentals/README.md

```diff
@@ -0,0 +1,169 @@
+
+**What should you do if you are working in a customer's API ecosystem where different design decisions have been made that may be out of line with the latest guidance?**
+
+Every situation is unique and should be a conversation with the customer.  Ideally that conversation lands in one of two categories:
+
+* The customer wants to adopt industry best practices.  Signal a departure from their existing patterns with a major version change to the API or starting a new service root.
+* The customer wants to retain their existing patterns for this and future development. Document the generalized patterns, if possible, and apply them consistently in your design.
+
+## HTTP Methods
+
+[source](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#74-supported-methods)
+
+> HTTP methods are frequently referred to as the HTTP verbs.
+The terms are synonymous in this context, however the HTTP specification uses the term method.
+
+Operations MUST use the proper HTTP methods whenever possible, and operation idempotency MUST be respected.  Not all resources will support all methods, but all resources using the methods below MUST conform to their usage.
+
+Method  | Description                                                                                                                | Is Idempotent
+------- | -------------------------------------------------------------------------------------------------------------------------- | -------------
+GET     | Return the current value of an object                                                                                      | True
+PUT     | Replace an object, or create a named object, when applicable                                                               | True
+DELETE  | Delete an object                                                                                                           | True
+POST    | Create a new object based on the data provided, or submit a command                                                        | False
+HEAD    | Return metadata of an object for a GET response. Resources that support the GET method MAY support the HEAD method as well | True
+PATCH   | Apply a partial update to an object                                                                                        | False
+OPTIONS | Get information about a request; see below for details.                                                                    | True
+
+## HTTP Status Codes
+
+The following HTTP status codes are used by the API. Additional codes might be added in the future and if you encounter a code not in this list please consult the [HTTP specification](http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10/) for a definition.
```

Comment:
This looks like it is perhaps copy/pasted from some documentation, as it is referring to a given API -- "The following HTTP status codes are used by the API." -- perhaps we can replace this with something like "The following are some HTTP status code commonly used by APIs"?